test/TestHelper.cpp

```diff
@@ -184,8 +271,17 @@ void ImportTest::exportTest(bytes const& _output, State const& _statePost)
 	// export logs
 	m_TestObject["logs"] = exportLog(_statePost.pending().size() ? _statePost.log(0) : LogEntries());
 
-	// export post state
+	// compare expected state with post state
+	if (m_TestObject.count("expect") > 0)
+	{
+		stateOptionsMap stateMap;
+		State expectState(OverlayDB(), eth::BaseState::Empty);
+		importState(m_TestObject["expect"].get_obj(), expectState, stateMap);
+		checkExpectedState(expectState, _statePost, stateMap, Options::get().checkState ? WhenError::Throw : WhenError::DontThrow);
```

Comment:
instead of stateMap, you can just do stateOptionsMap(), since you do not use stateMap otherwise, So you can delet line 277.